packages/client/wallets/smart-wallet/src/blockchain/wallets/EVMSmartWallet.ts

```diff
@@ -104,8 +104,6 @@ export class EVMSmartWallet extends LoggerWrapper {
      * @returns A list of NFTs owned by the wallet.
      */
     public async nfts() {
-        return this.logPerformance("GET_NFTS", async () => {
-            return this.crossmintService.fetchNFTs(this.accountClient.account.address, this.chain);
-        });
```

Comment:
This change snuck in, but deleting here since we already log the performance of every API call, so this is redundant.